misc.sk

```diff
@@ -10,6 +10,52 @@ mining:
 	emerald¦s = minecraft:emerald
 	flint¦s = minecraft:flint
 
+village and pillage update:
+	minecraft version = 1.14 or newer
+	{composter stage}:
+		{default} = -
+		(empty|unfilled|level 0) = -[level=0]
+		level 1 = -[level=1]
+		level 2 = -[level=2]
+		level 3 = -[level=3]
+		(half full|half empty|level 4) = -[level=4]
+		level 5 = -[level=5]
+		level 6 = -[level=6]
+		level 7 = -[level=7]
+		(filled|level 8) = -[level=8]
+	{composter stage} composter¦s = minecraft:composter
+
+	{bamboo age}:
+		{default} = -
+		(thin|age 0) = -[age=0]
+		(thick|age 1) = -[age=1]
+	{bamboo leaves}:
+		{default} = -
+		leafless = -[leaves=none]
+		small leaved -[leaves=small]
+		large leaved -[leaves=large]
+	{bamboo stage}:
+		{default} = -
+		(stagnant|stage 0) = -[stage=0]
+		(growing|stage 1) = -[stage=1]
+	{bamboo age} {bamboo leaves} {bamboo stage} bamboo [plant] = minecraft:bamboo
+	bamboo = minecraft:bamboo
+
+	{berry age}:
+		{default} = -
+		young = -[age=0]
+		berryless = -[age=1]
+		(some berries|berried) = -[age=2]
+		(many berries|very berried) = -[age=3]
+	{berry age} sweet berry bush [plant] = minecraft:sweet_berry_bush
+
+	flower charge banner pattern¦s = minecraft:flower_banner_pattern
+	creeper charge banner pattern¦s = minecraft:creeper_banner_pattern
+	skull charge banner pattern¦s = minecraft:skull_banner_pattern
+	(mojang|thing) banner pattern¦s = minecraft:mojang_banner_pattern
+	globe banner pattern¦s = minecraft:globe_banner_pattern
+	[any] banner pattern¦s = flower charge banner pattern, creeper charge banner pattern, skull charge banner pattern, thing banner pattern, globe banner pattern,
```

Comment:
Is the extra comma at the end of this line harmless? I actually have no idea how the aliases handle that.